src/coins.ts

```diff
@@ -258,9 +260,11 @@ export const coins = CoinMap.fromCoins([
   erc20('storm', 'Storm', 18, '0xd0a4b8946cb52f0661273bfbc6fd0e0c75fc6433', UnderlyingAsset.STORM),
   erc20('sub', 'Substratum', 2, '0x12480e24eb5bec1a9d4369cab6a80cad3c0a377a', UnderlyingAsset.SUB),
   erc20('taud', 'TrueAUD', 18, '0x00006100f7090010005f1bd7ae6122c3c2cf0090', UnderlyingAsset.TAUD),
+  erc20('tcad', 'TrueCAD', 18, '0x00000100f2a2bd000715001920eb70d229700085', UnderlyingAsset.TCAD),
```

Comment:
Interesting that there are so many null bytes in the TrueAUD/CAD/HKD contract addresses. Any idea what's behind that? It's really unlikely that all these have so many null bytes by chance. (don't spent too much time on it - I'm just curious).

Reply:
They grind for these addresses to make them easier to recognize

Reply:
https://blog.trusttoken.com/upgrade-80ba355f8960 "Why does the..."